java/src/test/java/org/hyperledger/fabric/client/FileCheckpointerTest.java

```diff
@@ -0,0 +1,55 @@
+package org.hyperledger.fabric.client;
+
+
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class FileCheckpointerTest extends CommonCheckpointerTest {
+    FileCheckpointer fileCheckpointer;
+    private static final TestUtils testUtils = TestUtils.getInstance();
+
+    @Override
+    protected FileCheckpointer getCheckpointerInstance() throws IOException {
+        Path file = testUtils.createTempFile(".json");
+        this.fileCheckpointer = new FileCheckpointer(file);
```

Comment:
No `this` unless it's necessary because of a shadowed variable name!
```suggestion
       fileCheckpointer = new FileCheckpointer(file);
```